# mission phases must be sent to planner incrementally
# i.e. phase 1 must be done completely before beginning phase 2.
# If more than 1 team of vehicles present, then populate array appropriately

fast_task_planner/MissionPlan[] missions